src/components/PerProjectChart.tsx

```diff
@@ -87,77 +139,86 @@ export const PerProjectChart = ({ entries, projectCount = 5, className }: PerPro
-                    {prettifyProgrammingLanguageName(l.slice(0, l.length - 9)) ?? "Unknown"}:{" "}
-                    {prettyDuration(Number(point.data[l]))}
-                  </List.Item>
-                ))}
-            </List>
-          </Paper>
-        )}
-        valueFormat={v => prettyDuration(v)}
-        layout="horizontal"
-        colors={{ scheme: "paired" }}
-        // TODO: Format the values properly instead of the raw "*_duration" format
-        legendLabel={datum => {
-          const str = String(datum.id);
-          return prettifyProgrammingLanguageName(str.slice(0, str.length - 9)) ?? "Unknown";
-        }}
-        legends={[
-          {
-            dataFrom: "keys",
-            anchor: "bottom-right",
-            direction: "column",
-            translateX: 160,
-            itemWidth: 80,
-            itemHeight: 20
-          }
-        ]}
-      />
+    <div className={className}>
+      <div style={{ height: 110 * Math.min(totalTimeByProject.length, projectCount), width: "100%" }}>
+        <ResponsiveBar
+          data={data}
+          keys={[...languageNames.map(l => `${l}_duration`)]}
+          labelSkipWidth={10}
+          indexBy="projectName"
+          margin={{
+            top: 30,
+            right: (window.innerWidth > 740 ? window.innerWidth * 0.05 : 0),
+            bottom: 30,
+            left: (window.innerWidth > 740 ? window.innerWidth * 0.05 : 0)
+             + (longestProjectName.length > 8 ? longestProjectName.length * 6 : 0)
+          }}
+          padding={0.3}
+          enableGridX
+          enableGridY={false}
+          maxValue={ticks[ticks.length - 1]}
+          theme={{ textColor: usesDarkMode ? "white" : "black" }}
+          axisBottom={{
+            format: (d: number) => prettyDuration(d),
+            tickValues: ticks
+          }}
+          borderRadius={2}
+          labelSkipHeight={20}
+          labelTextColor="black"
+          gridXValues={ticks}
+          axisLeft={{
+            tickPadding: 8
+          }}
+          tooltipLabel={d => String(d.data.projectName)}
+          tooltip={point => (
+            <Paper
+              sx={theme => ({
+                backgroundColor: usesDarkMode
+                  ? theme.colors.dark[5]
+                  : theme.colors.gray[1]
+              })}
+              p={10}
+            >
+              <Text underline>{point.label}</Text>
+              <List>
+                {Object.keys(point.data)
+                  .filter(k => k.endsWith("_duration"))
+                  .reverse()
+                  .map(l => (
+                    <List.Item key={l}>
+                      {prettifyProgrammingLanguageName(l.slice(0, l.length - 9)) ?? "Unknown"}:{" "}
+                      {prettyDuration(Number(point.data[l]))}
+                    </List.Item>
+                  ))}
+              </List>
+            </Paper>
+          )}
+          valueFormat={v => prettyDuration(v)}
+          layout="horizontal"
```

Comment:
Remove this commented line

Reply:
Umm?

Reply:
Oops, I think I selected the wrong line. I meant line 198 (`// colors={{ scheme: "paired" }}`)

Reply:
Oh! Alright, I'll remove that as well.